.github/workflows/ci.yaml

```diff
@@ -132,8 +171,9 @@ jobs:
 
         - name: Run tests
           run: |
+            sudo apt-get update && sudo apt-get install -y libopenmpi-dev
             pip install -e .
             cd test
-            python -m pip install pytest pymbolic psutil matplotlib
+            python -m pip install pytest pymbolic psutil matplotlib mpi4py
```

Comment:
Is MPI required for pytest?

Reply:
Not since the change to call into runalyzer libraries for testing gather